### DISCUSSION TITLE
How to delete an asset folder

### DISCUSSION THREAD

**Discussion by parthi2929** [requarks/wiki#3586] [Error / Bug Report]
**Describe the bug**
I am new wiki user and we host it in our VM. While editing a page, I accidentally created wrong assets folder. How could I delete them? I could not find any option going to assets option. 

**To Reproduce**
1. Go to assets option in editor page (markdown)
2. Create a folder
3. Try deleting it

**Expected behavior**
A clear option well placed to enable us to delete asset folders

**Screenshots**
Here I want to delete temperature folder which I created at same level as sensors, but it should be within sensors. 
![image](https://user-images.githubusercontent.com/11896263/87749300-f647f200-c815-11ea-9455-c28e47d3aec3.png)

**Host Info (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04, Docker]  Ubuntu 18.04
 - Wiki.js version: Wiki.js 2.4.105
 - Database engine: As requested by Wiki.js 2.4.105 (is this needed, I have to ask admin, if so I will raise request and get it)

**Comment by 3DSkaoi** [comment]
From the doc, https://docs.requarks.io/guide/assets, it is written that it's coming soon.

**Reply by inigma2** [reply]
this has been planned since 2019. It's now 2021. When?

**Reply by GesiIvy128** [reply]
And it's now 2023, close to 2024, and it's still 'coming soon'. LOL~

**Reply by chengpengzhao** [reply]
And it's now 2025, and it's still 'coming soon'. LOL~

**Reply by yusufgungor** [reply]
And it's now 2026, and it's still 'coming soon'. LOL~

**Comment by ProcrastinatingDev** [comment]
Currently performing asset folder cleanups through DB... 
Do you have DB access?

**Reply by inigma2** [reply]
> Currently performing asset folder cleanups through DB...
> Do you have DB access?

No I don't have db access.

**Comment by pwjpj52** [comment]
SO HOW TO DELETE IT FROM DATABASE

**Reply by v0tti** [reply]
List asset folders:
`SELECT * FROM "assetFolders";`

Delete asset folder:
`DELETE FROM "assetFolders" WHERE id=XX;`

Replace XX with the ID of the folder you want to delete. If you want to delete a folder with child folders, you first have to delete those.
